### DISCUSSION TITLE
How to exclude POI "Stolpersteine" from "Sehenswürdigkeiten" (sights/touristattraction)

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#15159] [Q&A] [answered]
Is ist possible, to exclude POI Stolpersteine (https://wiki.openstreetmap.org/wiki/DE:Stolpersteine) from search-results for sights ("Sehenswürdigkeiten")?
When I'm in a city thas has "Stolpersteine", there are maybe hundreds of them. Then its not easy to find a "real" sight.

**Comment by tommycrock** [comment, marked as answer]
Make a custom search (there are probably many other 'tourism' things you aren't interested in).
Search>Categories>Custom search>Tourism>tick the options that interest you and follow the prompts to save it.

**Reply by ThomasMeister123** [reply]
OK, thank you. But in this case I have to exclude ALL "memorials". But thats OK.
